Bio/Phylo/_utils.py

```diff
@@ -312,15 +312,15 @@ def draw(tree, label_func=str, do_show=True, show_confidence=True,
 
     Example using the pyplot options 'axhspan' and 'axvline':
 
-    >>> from Bio import Phylo, AlignIO
-    >>> from Bio.Phylo.TreeConstruction import DistanceCalculator, DistanceTreeConstructor
-    >>> constructor = DistanceTreeConstructor()
-    >>> aln = AlignIO.read(open('Tests/TreeConstruction/msa.phy'), 'phylip')
-    >>> calculator = DistanceCalculator('identity')
-    >>> dm = calculator.get_distance(aln)
-    >>> tree = constructor.upgma(dm)
-    >>> Phylo.draw(tree, axhspan=((0.25, 7.75), {'facecolor':'0.5'}),
-    ...     axvline={'x':'0', 'ymin':'0', 'ymax':'1'})
+    from Bio import Phylo, AlignIO
```

Comment:
Can you indent all of this four spaces, and end the previous line with a double colon - that gives an RST literal block. It will then render much better - try it on http://rst.ninjs.org or your favourite RST editor.